core/tests/protractor_utils/AdminPage.js

```diff
@@ -34,6 +34,13 @@ var AdminPage = function() {
   var roleSelect = element(by.css('.protractor-update-form-role-select'));
   var statusMessage = element(by.css('[ng-if="$ctrl.statusMessage"]'));
 
+  // Viewing roles can be done by two methods: 1. By roles 2. By username
+  var roleDropdown = element(by.css('.protractor-test-view-role-method'));
```

Comment:
Also remove view- from the protractor-test-* css class names (here and below)

Reply:
Done!